script.module.python.twitch/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.module.python.twitch" name="python-twitch for Kodi" version="1.0.2" provider-name="A Talented Community">
+<addon id="script.module.python.twitch" name="python-twitch for Kodi" version="1.1.0" provider-name="A Talented Community">
     <requires>
         <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
Please change python version to 2.20.0

Reply:
Thanks, updated xbmc.python version and squashed